std/numeric.d

```diff
@@ -2572,27 +2572,55 @@ GapWeightedSimilarityIncremental!(R, F) gapWeightedSimilarityIncremental(R, F)
 
 /**
 Computes the greatest common divisor of $(D a) and $(D b) by using
-Euclid's algorithm.
+Euclid's or Stein's algorithm depending on compiler.
  */
 T gcd(T)(T a, T b)
 {
-    static if (is(T == const) || is(T == immutable))
-    {
-        return gcd!(Unqual!T)(a, b);
-    }
-    else
-    {
-        static if (T.min < 0)
+    version(DigitalMars) {
```

Comment:
Since the CircleCI style checker has been re-enabled the [failing status](https://circleci.com/gh/dlang/phobos/923) of CircleCi refers to detected style violations :)
(or of course to build errors).

@Darredevil there's also a `style` target in the Makefile, so you can e.g. use `make -f posix.mak style` to run the linting locally.